src/Lessons-api.Data/Models/StudentEntity.cs

```diff
@@ -1,12 +1,17 @@
-﻿namespace Lessons_api.Data.Models
+﻿using System.Collections.Generic;
+
+namespace Lessons_api.Data.Models
 {
     public class StudentEntity
     {
+        public StudentEntity()
+        {
+            StudentLessonRels = new HashSet<StudentLessonRel>();
+        }
+
         public int Id { get; set; }
-        public string FirstName { get; set; }
-        public string LastName { get; set; }
-        public string Country { get; set; }
-        public string City { get; set; }
-        public int Age { get; set; }
+        public int UserId { get; set; }
+        public UserEntity User { get; set; }
+        public ICollection<StudentLessonRel> StudentLessonRels { get; set; }
```

Comment:
You can also add Collection of Lessons here.
Please check [this link]([this link](https://docs.microsoft.com/en-us/ef/core/modeling/relationships?tabs=fluent-api%2Cfluent-api-simple-key%2Csimple-key)) for reference